openid-connect-key-binding-1_0.md

```diff
@@ -38,11 +38,11 @@ OpenID Key Binding specifies how to bind a public key to an OpenID Connect ID To
 
 OpenID Connect is a protocol that enables a Relying Party (RP) to delegate authentication and obtain identity claims to an OpenID Connect Provider (OP).
 
-When authenticating with OpenID Connect, RP provides a nonce in its authentication request. The ID Token signed and returned by the OP contains the nonce and claims about the user. When verifying the ID Token, the RP confirms it contains the nonce, binding the session that made the request to the response.
+When authenticating with OpenID Connect, an RP provides a nonce in its authentication request. The ID Token signed and returned by the OP contains the nonce and claims about the user. When verifying the ID Token, the RP confirms it contains the nonce, binding the session that made the request to the response.
 
-When an RP wants to prove to another system that it has authenticated a user, it may present the ID Token as a bearer token. However, bearer tokens are vulnerable to theft and replay attacks - if an attacker intercepts the ID Token, they can impersonate the authenticated user to downstream systems that accept a ID Token as a bearer token.
+It is common for an RP to be composed of multiple components. When the RP authenticating component that obtained the ID Token wants to prove to an RP consuming component that it has authenticated a user, it may present the ID Token as a bearer token. However, bearer tokens are vulnerable to theft and replay attacks - if an attacker obtains the ID Token, they can impersonate the authenticated user.
```

Comment:
This authenticating/consuming terminology I am unfamiliar with.

What is the RP authenticating component vs the RP consuming component? What are the RP consuming components consuming?  Wouldn't it simply be that an RP consists of a set of authenticating components?

This is my attempt rewrite this to provide clarity on how I think about it.

> It is common for an RP to be composed of multiple components. When the RP component that obtained the ID Token wants to prove to another RP component that it has authenticated a user, it may present the ID Token as a bearer token. However, bearer tokens are vulnerable to theft and replay attacks - if an attacker obtains the ID Token **for instance by compromising one these components**, they can impersonate the authenticated user.